pkg/authz/oauth/clientmetadata.go

```diff
@@ -73,8 +73,8 @@ type clientMetadata struct {
 	TLSClientAuthSanIP      string                 `json:"tls_client_auth_san_ip,omitempty"`
 	TLSClientAuthSanURI     string                 `json:"tls_client_auth_san_uri,omitempty"`
 	RegistrationAccessToken string                 `json:"registration_access_token,omitempty"`
+	ExtraProperties         map[string]interface{} `json:"-"`
```

Comment:
do not see the need to export this yet, as its not used by the json marshal call or outside of this package.